src/features/profile/Profile.tsx

```diff
@@ -76,9 +77,7 @@ const profile: React.FC<Props> = ({
           </Text>
           <RadioButton
             isChecked={theme === Themes.light}
-            onPress={() => {
-              setTheme(Themes.light);
-            }}
+            onPress={() => setTheme(Themes.light)}
```

Comment:
You can define 2 callbacks with diff args or put there condition and use 1 callback.